admin/controllers/tech-support.js

```diff
@@ -507,6 +508,49 @@ const controller = {
     } catch (error) {
       return next(error)
     }
+  },
+
+  getPsReportLogs: async function getPsReportLogs (req, res, next) {
+    try {
+      const logs = await psReportLogsDownloadService.getLogFoldersList()
+      res.locals.pageTitle = 'PS Report Logs'
+      req.breadcrumbs('PS Report Logs')
+      res.render('tech-support/ps-report-logs', {
+        breadcrumbs: req.breadcrumbs(),
+        logs: logs
+      })
+    } catch (error) {
+      return next(error)
+    }
+  },
+
+  getPsReportLogsFileList: async function getPsReportLogsFileList (req, res, next) {
```

Comment:
Suggest we rename `getPsReportLogsFileList` to `getPsReportFolderFileList` as I admit I was a bit confused on reading this as to the difference between this page and the previous one.